tests/server/routers/test_versions.py

```diff
@@ -0,0 +1,19 @@
+# pylint: disable=relative-beyond-top-level
+import unittest
+
+from optimade import __api_version__
+from ..utils import SimpleEndpointTestsMixin
```

Comment:
```suggestion
from optimade.server.utils import SimpleEndpointTestsMixin
```

Reply:
The import of test utils module, so this shouldn't be changed